integration-libs/opf/base/components/opf-quick-buy/apple-pay/apple-pay.component.ts

```diff
@@ -52,6 +53,9 @@ export class ApplePayComponent implements OnInit {
   isApplePaySupported$: Observable<boolean>;
   applePayDigitalWallet?: DigitalWalletQuickBuy;
 
+  ngOnDestroy(): void {
+    this.cartHandlerService.allowMiniCartUpdate();
```

Comment:
we want to ensure the cart does not stay blocked. Even though the logic should be handled, we want to avoid any obscured scenario.